static/schemas/2.0/user.2.0.json

```diff
@@ -0,0 +1,30 @@
+{
+    "title": "User.2.0",
+    "definitions": {
+        "PartialUser": {
+            "description": "A RackHD local user",
+            "type": "object",
+            "properties": {
+                "username": {
+                    "description": "The username of the user",
+                    "type": "string"
+                },
+                "password": {
+                    "description": "The password of the user",
+                    "type": "string"
```

Comment:
Recommend to add more conditions for `username` and `password`, these conditions will be played as validation code and keep the code & data clean.  Such as use `minLength` to specify the minimum characters for that filed, and use `patter` to screen some invalid characters.